examples/sample-layers/plot-layer/label-vertex.glsl

```diff
@@ -115,6 +114,6 @@ void main(void) {
   label_vertex *= fontSize / labelHeight * position_clipspace.w;
 
   gl_Position = position_clipspace + 
-    vec4(label_vertex / screenSize, 0.0, 0.0);
+    vec4(label_vertex / viewportSize, 0.0, 0.0);
```

Comment:
Not sure about this one, but maybe there is a better naming here. 

`label_point`?  Seemed strange to divide a vertex by a `vec2` until I realized `label_vertex` is a point.